lockbox-ios/Store/AccountStore.swift

```diff
@@ -80,6 +80,7 @@ class AccountStore: BaseAccountStore {
     override func initialized() {
         self.dispatcher.register
                 .filterByType(class: AccountAction.self)
+                .observeOn(MainScheduler.instance)
```

Comment:
This seems like the simplest fix.

Are the other actions (`oauthLogin`, `oauthSignInMessageRead`) expensive or do any IO traffic? If so, then perhaps an `async` call may be necessary.

Reply:
For the other actions: `oathLogin` calls some asynchronous code that runs on a background thread and `clearOldKeychainValues` needs to be on the main thread because it also calls Webkit functions.